### PR TITLE
Fixed PEP8 issues except E501.

### PR DESCRIPTION
**Pull request by blackpioter**
This one is in regard to Issue #464.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=466) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=466) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-python?pullRequest=466) it.</sub>

File: examples/clients/clients.py [hunk 1]
```diff
@@ -9,7 +9,8 @@
 # Retrieve email statistics by client type. #
 # GET /clients/stats #
 
-params = {'aggregated_by': 'day', 'start_date': '2016-01-01', 'end_date': '2016-04-01'}
+params = {'aggregated_by': 'day',
```
Comment (mbernier) [inline examples/clients/clients.py]: Would it be ok to be pedantic here and ask that end_date be moved to the next line too?

File: examples/clients/clients.py [hunk 2]
```diff
@@ -19,10 +20,10 @@
 # Retrieve stats by a specific client type. #
 # GET /clients/{client_type}/stats #
 
-params = {'aggregated_by': 'day', 'start_date': '2016-01-01', 'end_date': '2016-04-01'}
+params = {'aggregated_by': 'day',
```
Comment (mbernier) [inline examples/clients/clients.py]: Would it be ok to be pedantic here and ask that end_date be moved to the next line too?

File: examples/contactdb/contactdb.py [hunk 3]
```diff
@@ -88,11 +88,12 @@
...
 
 data = {
-  "name": "newlistname"
+    "name": "newlistname"
 }
 params = {'list_id': 1}
 list_id = "test_url_param"
-response = sg.client.contactdb.lists._(list_id).patch(request_body=data, query_params=params)
+response = sg.client.contactdb.lists._(list_id).patch(
+    request_body=data, query_params=params)
```
Comment (mbernier) [inline examples/contactdb/contactdb.py]: Can you please move query_params to the next line?

File: examples/contactdb/contactdb.py [hunk 4]
```diff
@@ -343,20 +348,21 @@
...
+        }
+    ],
+    "list_id": 5,
+    "name": "The Millers"
 }
 params = {'segment_id': 'test_string'}
 segment_id = "test_url_param"
-response = sg.client.contactdb.segments._(segment_id).patch(request_body=data, query_params=params)
+response = sg.client.contactdb.segments._(segment_id).patch(
+    request_body=data, query_params=params)
```
Comment (mbernier) [inline examples/contactdb/contactdb.py]: Can you please move query_params to the next line?

File: examples/devices/devices.py [hunk 5]
```diff
@@ -9,9 +9,9 @@
 # Retrieve email statistics by device type. #
 # GET /devices/stats #
 
-params = {'aggregated_by': 'day', 'limit': 1, 'start_date': '2016-01-01', 'end_date': '2016-04-01', 'offset': 1}
+params = {'aggregated_by': 'day', 'limit': 1,
+          'start_date': '2016-01-01', 'end_date': '2016-04-01', 'offset': 1}
```
Comment (mbernier) [inline examples/devices/devices.py]: Can you please move end_date and offset to their own lines?

File: examples/geo/geo.py [hunk 6]
```diff
@@ -9,9 +9,9 @@
 # Retrieve email statistics by country and state/province. #
 # GET /geo/stats #
 
-params = {'end_date': '2016-04-01', 'country': 'US', 'aggregated_by': 'day', 'limit': 1, 'offset': 1, 'start_date': '2016-01-01'}
+params = {'end_date': '2016-04-01', 'country': 'US', 'aggregated_by': 'day',
+          'limit': 1, 'offset': 1, 'start_date': '2016-01-01'}
```
Comment (mbernier) [inline examples/geo/geo.py]: can you please move the params to each be on their own line?

File: examples/helpers/mail/mail_example.py [hunk 7]
```diff
@@ -104,22 +108,28 @@ def build_kitchen_sink():
     mail_settings = MailSettings()
     mail_settings.bcc_settings = BCCSettings(True, Email("test@example.com"))
     mail_settings.bypass_list_management = BypassListManagement(True)
-    mail_settings.footer_settings = FooterSettings(True, "Footer Text", "<html><body>Footer Text</body></html>")
+    mail_settings.footer_settings = FooterSettings(
```
Comment (mbernier) [inline examples/helpers/mail/mail_example.py]: can you please move the params to each be on their own line?

File: examples/clients/clients.py (hunk 1 above)
Comment (blackpioter) [reply examples/clients/clients.py]: Updated.

File: examples/clients/clients.py (hunk 2 above)
Comment (blackpioter) [reply examples/clients/clients.py]: Updated.

File: examples/helpers/mail/mail_example.py (hunk 7 above)
Comment (blackpioter) [reply examples/helpers/mail/mail_example.py]: Updated.

**Comment by blackpioter** [comment]
@mbernier I've updated PR according to comments. Let me know if you got some more ideas.

**Comment by mbernier** [comment]
Nice! Thank you so much!

**Comment by mbernier** [comment]
Had some merge conflicts. I believe that I resolved them properly. I am letting the build run to verify.

**Comment by blackpioter** [comment]
@mbernier cool. Build seems to be clean after conflicts resolved.

**Comment by thinkingserious** [comment]
@blackpioter

We have not been able to merge your Pull Request, but because you are awesome - we wanted to make sure you could still get a SendGrid Hacktoberfest shirt.  

Please go **fill [out our swag form](https://dx.sendgrid.com/swag) before Nov 5th** and we will send the shirt! (We know that you might have tried this before and it didn’t work, sorry about that!)

### **You have till Nov 5th to fill out this form** in order to get the Hacktoberfest shirt!

Thank you for contributing during Hacktoberfest! We hope to see you in the repos soon! Just so you know, we always give away a SendGrid shirt for your first ever non-Hacktoberfest PR that gets merged.

**Comment by thinkingserious** [comment]
Hello @blackpioter,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**